project-base/src/Shopsys/ShopBundle/Resources/views/Styleguide/styleguide.html.twig

```diff
@@ -0,0 +1,1126 @@
+{% extends '@ShopsysShop/Front/Layout/base.html.twig' %}
+
+{% block title %}Shopsys Framework styleguide{% endblock %}
+
+{% block meta_robots %}
+    <link rel="stylesheet" type="text/css" href="{{ asset('assets/styleguide/styles/styleguide_' ~ getCssVersion() ~ '.css') }}" media="screen, projection">
+{% endblock %}
+
+{% block javascripts %}
+    <!-- codemirror -->
+    {{ importJavascripts([
+        'styleguide/styleguide.js',
+        'styleguide/plugins/codemirror/codemirror.minified.js'
+    ]) }}
+
+{% endblock %}
+
+{% block html_body %}
```

Comment:
Maybe, I would like prefer split this template to few smaller templates by categories. For better readability and post editing.

Reply:
It would be great but I would maybe change it so it uses blocks from SSFW.. in the future. I'll create an issue or US for this